### DISCUSSION TITLE
V8时代的难题，看看爽局如何通关

### DISCUSSION THREAD

**Discussion by K-k423** [share121/inter-knot#4915] [崩坏：星穹铁道]
有很多人都在问怎么通关差分宇宙V8难度，以下是小编整理好的方法，跟着小编一起来看看吧
第三间拿到三个鲁伯特合成大爹，大爹在从牌堆中通常召唤出袍子，与死重逢和虚高一丈，通过掏垃圾从墓地中召唤出智齿，后场盖上两张大乐透，使用剪刀翻开移到前场，接着通过在冒险关里从30个罐子中召唤出分裂金币和死水书，最后用碎片买通两位天才，获得大量物品。
怎么样，都学废了吗![Screenshot_2024-09-15-21-38-33-904_com miHoYo hkrpg](https://github.com/user-attachments/assets/b5c85386-e814-4496-b529-17a73cf1a79f)

**Comment by shidijun** [comment]
东姐好<img src="https://img-static.mihoyo.com/communityweb/upload/81ed7cd6675af50aec9cc622bda99b88.png" alt="比利_害羞" width="75" >
